src/garage/np/algos/meta_rl_algorithm.py

```diff
@@ -0,0 +1,35 @@
+"""Interface of Meta-RL ALgorithms."""
+import abc
+
+from garage.np.algos import RLAlgorithm
+
+
+class MetaRLAlgorithm(RLAlgorithm, abc.ABC):
+    """Base class for Meta-RL Algorithms."""
+
+    @abc.abstractmethod
+    def get_exploration_policy(self):
+        """Return a policy used before adaptation to a specific task.
+
+        Each time it is retrieved, this policy should only be evaluated in one
+        task.
+
+        Returns:
+            garage.Policy: The "un-adapted" policy.
+
+        """
+
+    @abc.abstractmethod
+    def adapt_policy(self, exploration_policy, trajectories):
```

Comment:
perhaps this should be called `exploration_trajectories` or `new_task_trajectories`